aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1369,6 +1370,236 @@ template at::Tensor PackedConvWeightsOnednn<3>::apply_relu(
     double output_scale,
     int64_t output_zero_point);
 
+template <PostOps postOpFused>
+static at::Tensor _quantized_convolution_pt2e(
+    at::Tensor act, // contains quantized values but not QTensor
+    double act_scale,
+    int64_t act_zero_point,
+    at::Tensor weight, // MKLDNN tensor with quantized values
+    at::Tensor weight_scales,
+    at::Tensor weight_zero_points,
+    c10::optional<at::Tensor> bias, // Bias is not packed into MKLDNN tensor
+    torch::List<int64_t> stride,
+    torch::List<int64_t> padding,
+    torch::List<int64_t> dilation,
+    bool transposed,
+    int64_t groups,
+    double output_scale,
+    int64_t output_zero_point,
+    c10::optional<at::Tensor> accum, // accum to fused with conv add
+    double accum_scale,
+    int64_t accum_zero_point,
+    bool fp32_output,
+    const c10::optional<c10::ArrayRef<c10::IValue>>& post_op_args) {
+  /*********************************/
+  /*          Checks               */
+  /*********************************/
+
+  if (fp32_output) {
+    // when fp32_output, if we set op_attr.set_zero_points, oneDNN will get wrong result
```

Comment:
I guess this is an API protocol of oneDNN? If so, better to say "oneDNN expects ..." instead of "wrong result" which sounds like a bug.

Reply:
Thanks for the suggestions and changed.